test/test_checkpoint.py

```diff
@@ -0,0 +1,62 @@
+import torch
+import torch.nn as nn
+import math, unittest
+from torch.autograd import Variable
+from torch.utils.checkpoint import checkpoint_sequential
+from common import TestCase, run_tests
+
+
+class TestCheckpoint(TestCase):
```

Comment:
Can you move that to `test_autograd.py` or `test_utils.py`? We keep the tests organized by module